tech/database/mongodb/about.md

```diff
@@ -13,8 +13,9 @@ MongoDB is a free and open source database and uses a document-oriented data mod
 ## Installation 
 
 ```
-# dnf install mongodb mongodb-server
-# service mongod start
+# dnf install -y mongodb-org
+# systemctl enable mongod.service
```

Comment:
```suggestion
$ sudo systemctl enable mongod
```
Also the line above is missing the explanation, as mentioned in [writing tips](https://developer-portal.github.io/contributing/write-general-writing-hints.html).